eap6-support/hawkular-javaagent-wildfly-feature-pack-eap6/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -551,10 +552,6 @@ resource-type-set-dmr:
       attribute: product-name
     - name: Version
       attribute: product-version
-    - name: Running Mode
-      attribute: running-mode
-    - name: Server State
-      attribute: server-state
```

Comment:
need to put these back in. these do exist and IIRC MiQ uses them. I get this in my EAP 6.4 server:

/:read-attribute(name=server-state)
{
    "outcome" => "success",
    "result" => "running"
}

/:read-attribute(name=running-mode)
{
    "outcome" => "success",
    "result" => "NORMAL"
}

/:read-attribute(name=product-version)
{
    "outcome" => "success",
    "result" => "6.4.0.GA"
}

Reply:
Ah, I've tested in 6.4.7 and I didn't see them, I will double check.
Yes, if they are in 6.4 it should be back.
Thanks for checking it.